dev-go/go-md2man/go_md2man-1.0.7.recipe

```diff
@@ -0,0 +1,60 @@
+SUMMARY="Uses blackfriday to process markdown into man pages"
+DESCRIPTION="go-md2man uses blackfriday to process markdown into man pages"
```

Comment:
And here's a better `DESCRIPTION` as well (from [go-md2man.1](https://github.com/cpuguy83/go-md2man/blob/master/go-md2man.1.md))

>go-md2man converts standard markdown formatted documents into manpages. It is written purely in Go so as to reduce dependencies on 3rd party libs.